CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ For older changes see the [archived Singularity change log](https://github.com/a
   Unfortunately the feature does not work with "docker-archive" or "docker-daemon".
 - Change the default `arm` variant to `v7`, and stop using the GOARM environment
   variable. The variables GOOS, GOARCH and GOARM are only used when building.
+  The full buildkit log file is included in the image, for traceability.
+  It is also showed on the console, as a progress update while building.
```

Comment:
Looks like the beginning of the changelog entry got lost here.

Reply:
Probably a victim of the rebase, when the code was shuffled.